src/ucp/api/ucp.h

```diff
@@ -538,6 +538,22 @@ void ucp_worker_destroy(ucp_worker_h worker);
 
 
 /**
+ * @ingroup UCP_CONTEXT
+ * @bried Print information about protocols that would be used by a worker.
+ *
+ * This routine prints information about the protocols being used, thresholds,
+ * UCT methods, and so on, used by the current context.
+ *
+ * @param [in] context      Handle to @ref ucp_context_h "UCP application context".
+ * @param [in] stream       Output stream to print the information to.
+ * @param [in] title        Configuration title to print.
```

Comment:
it was simpler to define just print interface than query interface for now, for ucx_info.